h3/src/proto/frame.rs

```diff
@@ -62,11 +75,21 @@ impl From<usize> for PayloadLen {
 }
 
 impl Frame<PayloadLen> {
-    pub const MAX_ENCODED_SIZE: usize = VarInt::MAX_SIZE * 3;
+    pub const MAX_ENCODED_SIZE: usize = VarInt::MAX_SIZE * 7;
```

Comment:
This change might be obvious, sorry to ask, but this seems like a big jump. Is it spec related? (I actually have completely forgotten what this associated constant is really for...)

Reply:
It dictates the size of the WriteBuf which is used to encode the settings frame.

The previous limit was too small and panicked.

We need to send: `H3_DATAGRAM`, `H3_ENABLE_WEBTRANSPORT`, `ENABLE_CONNECT_PROTOCOL`, and `WEBTRANSPORT_SESSIONS` ontop of what is already sent in the settings frame